cmd/stats.go

```diff
@@ -46,8 +48,26 @@ var statsCmd = &cobra.Command{
 
 		// get values
 		maxDepth := len(longestChain)
-		directDeps := len(depGraph.Graph[depGraph.MainModuleName])
-		totalDeps := len(getAllDeps(depGraph.Graph[depGraph.MainModuleName], depGraph.TransDepList))
+		directDeps := 0
+		if len(mainModules) == 0 {
+			directDeps = len(depGraph.Graph[depGraph.MainModuleName])
+		} else {
+			var dirDeps []string
+			for _, mainModule := range mainModules {
+				dirDeps = getAllDeps(dirDeps, depGraph.Graph[mainModule])
+			}
+			directDeps = len(dirDeps)
+		}
```

Comment:
I think it would make more sense to accumulate a DirectDepList inside `getDepInfo` based on the mainModules passed in